### PR TITLE
Linting package docker

### PR DESCRIPTION
**Pull request by vdemeester**
Made a `golint` pass on package `docker` 🐙.

Few changes and a lot of comments. Remove one method that feels like it's not used or needed (at least in `libcompose`).

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: docker/name.go [hunk 1]
```diff
@@ -24,17 +25,13 @@ type singleNamer struct {
 	name string
 }
 
-func OneName(client dockerclient.Client, project, service string) (string, error) {
```
Comment (ibuildthecloud) [inline docker/name.go]: Was this function not used?

Comment (vdemeester) [inline docker/name.go]: Yes, not used anywhere :sweat_smile:

**Comment by ibuildthecloud** [comment]
LGTM

**Comment by vdemeester** [comment]
ping @aduermael @gdevillele

**Comment by dnephin** [comment]
LGTM, needs a rebase though

**Comment by vdemeester** [comment]
@dnephin rebased :wink:

**Pull request merged**